### ISSUE TITLE
Textures are flipped by Y-Axis 

### ISSUE THREAD

**Issue by stevan-voste** [atteneder/glTFast#711]
I’m encountering an issue with an imported GLTF model that uses KTX textures on an iOS device. 
The textures appear to be flipped along the Y-axis, but this only occurs on the device; everything works fine in the editor.

Note: I am using URP, and shaders are imported via shader variants 

**Files**
glb file to test 
[Attach or link to *.gltf/*.glb files that trigger the bug. ](https://myty-seed.ams3.cdn.digitaloceanspaces.com/models/bc449c03-91e5-43d4-a030-75a3d2a7af62/model-files/ktx.glb)

**To Reproduce**
1. Create empty Unity project with URP
2. Import provided model  
3. Build project on iOS device 

**Expected behavior**
Model textures are not flipped 

**Screenshots**
In application 
<img width="425" alt="image" src="https://github.com/user-attachments/assets/fcc45697-fb2c-4b62-bb90-cd7714b08619">

In editor 
![image](https://github.com/user-attachments/assets/b67f2055-fca1-4049-b26e-af30d5b2c583)

 - glTFast version 6.7.1
 - ktx: version 3.4.1
 - Unity Editor version 2022.3.41
 -  Universal Render Pipeline 14.0.11
 - iOS
 

 - Device:  iPhone14 Pro
 - OS : iOS 17.5.1

**Comment by atteneder** [comment]
Thanks for reporting!

To certify, you're not loading said model at runtime, but you've imported it (placed it in the *Assets* folder), right?

**Comment by atteneder** [comment]
I assumed that you imported at design-time and tried to reproduce.

TL;DR: Don't use KTX for this use case, but go with regular PNG/JPGs in glTF.

I think you entered unexplored territory there. The KTX for Unity plugin can import those bundled KTX textures, but it just transcodes them to a format your host computer supports (the machine you develop on). So my first attempt to re-produce failed, because on the Intel Mac it was transcoded to DXT, which is not supported on my iPhone. On an ARM Mac, it is transcoded to ETC and I did not see a problem.

You could try to figure out, which format you actually end up with.

But overall, this KTX workflow does not make sense to me at all.

**Comment by atteneder** [comment]
There's two potential problems underneath:

1. Incorrect texture format (dev host target format instead of generic format). See #552 .
2. Incorrect KTX texture orientation.

I don't assume it's the latter, but in case it is, it's outside the [extension spec](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_texture_basisu/README.md) and won't be fixed anytime soon.

> Orientation metadata (KTXorientation) MUST be rd or omitted.

Closing this now. Feel free to re-open if you need further assistance.
